.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    // "eslint:recommended",
+    // "plugin:@typescript-eslint/recommended",
+    // "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier",
+    // "prettier/@typescript-eslint",
```

Comment:
please remove comments

Reply:
Done - latest changes do not have any commented out rulesets in the ESLint config file